components/RepaymentInfo/RepaymentInfo.tsx

```diff
@@ -14,6 +17,36 @@ export function RepaymentInfo({ loan }: RepaymentInfoProps) {
     longFormattedInterestRate,
     longFormattedEstimatedPaybackAtMaturity,
   } = useLoanDetails(loan);
+
+  const maturityDate = useMemo(
+    () =>
+      loan.endDateTimestamp
+        ? dayjs.unix(loan.endDateTimestamp).format('MMMM D, YYYY')
+        : '',
+    [loan],
+  );
+
+  const createCalEvent = useCallback(() => {
+    const config: CalendarOptions = {
+      title: `Backed: Loan #${loan.id} ${loan.collateralName} due`,
+      description: `Loan repayment of ${longFormattedEstimatedPaybackAtMaturity} is due: https://www.withbacked.xyz/loans/${loan.id}`,
+      start: new Date(loan.endDateTimestamp! * 1000),
+    };
+
+    if (
```

Comment:
@adamgobes so if I understand correctly you're going to remove this conditional and just do `googleCalendar`?